commons/phone/phone_bluetooth_devices.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneBluetoothDevices",
+  "doc": "Phone Bluetooth devices info.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "pairedDevices", "type": "int", "doc": "Number of paired devices"},
+    {"name": "nearbyDevices", "type": "int", "doc": "Number of nearby devices"},
+    {"name": "enabled", "type": "boolean", "doc": "whether Bluetooth is enabled"}
```

Comment:
perhaps change the name to `bluetoothEnabled`?

Reply:
`bluetoothEnabled` would be indeed a better name, but you have already merged RADAR-Android-Phone branch which uses `enabled`. Can we maybe go with `enabled`, otherwise a new merge request for RADAR-Android-Phone would be needed?

Reply:
Done